pybamm/expression_tree/independent_variable.py

```diff
@@ -131,6 +131,12 @@ def __init__(
             raise pybamm.DomainError(
                 "domain cannot be particle if name is '{}'".format(name)
             )
+        # check coord_sys against KNOWN_COORD_SYS
+        if coord_sys in KNOWN_COORD_SYS:
+            pass
+        else:
+            raise ValueError(
+                f"Coordinate system is {coord_sys}, not in cartesian, cylindrical polar, spherical polar")
```

Comment:
I would reverse this:
```suggestion
        if coord_sys not in KNOWN_COORD_SYS:
            raise ValueError(
                f"Coordinate system is {coord_sys}, not in cartesian, cylindrical polar, spherical polar")
```